package.nls.json

```diff
@@ -436,7 +436,8 @@
     "DataScience.useCDNForWidgets": "Widgets require us to download supporting files from a 3rd party website. Click [here](https://aka.ms/PVSCIPyWidgets) for more information.",
```

Comment:
This message is already shown with three buttons, I didn't want to add another one, but I can if folks think that would be better.